nextflow.config

```diff
@@ -63,6 +78,11 @@ if(!params.igenomesIgnore){
   includeConfig 'conf/igenomes.config'
 }
 
+// Avoid this error:
+// WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
+// Thanks to: https://github.com/alesssia/YAMP/wiki/How-to-use-Docker
+docker.runOptions = '-u $(id -u):$(id -g)'
```

Comment:
```suggestion
```

Reply:
Yeah, I guess its just something I'll probably add to our template code now to have it in the next template release synced to all pipelines 👍